alibi_detect/cd/tabular.py

```diff
@@ -146,27 +148,3 @@ def _get_counts(self, x: np.ndarray, categories: Dict[int, List[int]]) -> Dict[i
         Utility method for getting the counts of categories for each categorical variable.
         """
         return {f: [(x[:, f] == v).sum() for v in vals] for f, vals in categories.items()}
-
-    def get_config(self) -> dict:
-        """
-        Get the detector's configuration dictionary.
-
-        Returns
-        -------
-        The detector's configuration dictionary.
-        """
-        cfg = super().get_config()
-
-        # Prep categories_per_feature. NOTE - toml can't write dict with int keys. See loading.read_detector_config
```

Comment:
Where is this done now that it's not done here?

Reply:
Its done in `_int2str_keys` in `saving.saving` ([here](https://github.com/ascillitoe/alibi-detect/blob/ab3562fc2ade48cf88eb8a81dd38e35b33602925/alibi_detect/saving/saving.py#L226)).

At load time, `pydantic` does the reverse for us (coerces `str` keys back to `int`), since we have the following in the `pydantic` models:
```python
categories_per_feature: Dict[int, Union[int, List[int]]] = None
```

Reply:
Its done in `_int2str_keys` in `saving.saving` ([here](https://github.com/ascillitoe/alibi-detect/blob/ab3562fc2ade48cf88eb8a81dd38e35b33602925/alibi_detect/saving/saving.py#L226)).

At load time, `pydantic` does the reverse for us (coerces `str` keys back to `int`), since we have the following in the `pydantic` models:
```python
categories_per_feature: Dict[int, Union[int, List[int]]] = None
```